src/shared/components/query/CancerStudyTreeData.ts

```diff
@@ -5,17 +5,21 @@ import {VirtualStudy} from "shared/model/VirtualStudy";
 
 export const CANCER_TYPE_ROOT = 'tissue';
 export const VIRTUAL_STUDY_NAME= 'My Virtual Studies';

```

Comment:
Should we use the Pluralize to deal with the study/studies?

Reply:
No, I think it would not be right to have a cateogory "My Virtual Study" which would seem to convey "you only get one!".    Just like the github Issues tab is not Issue if there's only one issue.